storagenode/storagenodedb/infodb.go

```diff
@@ -405,6 +405,52 @@ func (db *InfoDB) Migration() *migrate.Migration {
 					return nil
 				}),
 			},
+			{
+				Description: "Add reputation and storage usage cache tables",
+				Version:     15,
+				Action: migrate.SQL{
+					`CREATE TABLE reputation (
+						satellite_id BLOB NOT NULL,
+						uptime_success_count INTEGER NOT NULL,
+						uptime_total_count INTEGER NOT NULL,
+						uptime_reputation_alpha REAL NOT NULL,
+						uptime_reputation_beta REAL NOT NULL,
+						uptime_reputation_score REAL NOT NULL,
+						audit_success_count INTEGER NOT NULL,
+						audit_total_count INTEGER NOT NULL,
+						audit_reputation_alpha REAL NOT NULL,
+						audit_reputation_beta REAL NOT NULL,
+						audit_reputation_score REAL NOT NULL,
+						updated_at TIMESTAMP NOT NULL,
+						PRIMARY KEY (satellite_id)
```

Comment:
wouldnt it make sense to have a primary key as (satellite_id, updated_at) here

Reply:
We don’t care about past entries, if I understand correctly

Reply:
okay, so its basically just a single entry per satellite?

Reply:
Yeah